packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -1757,13 +1757,7 @@ class _CupertinoDatePickerMonthYearState extends State<CupertinoDatePicker> {
     double totalColumnWidths = 3 * _kDatePickerPadSize;
 
     for (int i = 0; i < columnWidths.length; i++) {
-      late final double offAxisFraction;
-      switch (i) {
-        case 0:
-          offAxisFraction = -0.3 * textDirectionFactor;
-        default:
-          offAxisFraction = 0.5 * textDirectionFactor;
-      }
+      final double offAxisFraction = (i == 0 ? -0.3 : 0.5) * textDirectionFactor;
```

Comment:
IMO this is not easier to understand. We could simply use a regular switch expression instead